src/helpers/buffers/unordered_receiver.rs

```diff
@@ -381,14 +381,14 @@ mod test {
                 let recv = recv.clone();
                 async move {
                     let f: Fp31 = recv.recv(0_usize).await.unwrap();
-                    assert_eq!(f, Fp31::from(18_u128));
+                    assert_eq!(f, Fp31::try_from(18).unwrap());
                 }
             }),
             spawn({
                 let recv = recv.clone();
                 async move {
                     let f: Fp32BitPrime = recv.recv(1_usize).await.unwrap();
-                    assert_eq!(f, Fp32BitPrime::from(0x0100_020c_u128));
+                    assert_eq!(f, Fp32BitPrime::try_from(0x0100_020c).unwrap());
```

Comment:
Why know these will not truncate.

Reply:
This one can be `truncate_from`. 0x0100_020C = 16,777,740 < Fp32BitPrime::PRIME